composer.json

```diff
@@ -1,14 +0,0 @@
-{
```

Comment:
Yeah, that's fine. I thought the Drupal composer system deferred to modules that had one. Maybe that's just in the future.